e2e_tests/tests/cluster/test_webhooks.py

```diff
@@ -149,3 +151,134 @@ def test_log_pattern_send_webhook(should_match: bool) -> None:
     else:
         assert default_path not in responses
         assert slack_path not in responses
+
+
+def create_default_webhook(sess: api.Session, workspaceId: Optional[int] = None) -> int:
+    webhook_trigger = bindings.v1Trigger(
+        triggerType=bindings.v1TriggerType.EXPERIMENT_STATE_CHANGE,
+        condition={"state": "COMPLETED"},
+    )
+    webhook_url = "http://localhost"
+    res = bindings.post_PostWebhook(
+        sess,
+        body=bindings.v1Webhook(
+            url=webhook_url,
+            webhookType=bindings.v1WebhookType.DEFAULT,
+            triggers=[webhook_trigger],
+            mode=bindings.v1WebhookMode.WORKSPACE,
+            name="",
+            workspaceId=workspaceId,
+        ),
+    )
+    return res.webhook.id or 0
+
+
+@pytest.mark.e2e_cpu
+def test_webhook_permission() -> None:
+    # non-admin should not be able to create global webhook.
+    user1_sess = api_utils.user_session()
+    with pytest.raises(errors.APIException):
+        create_default_webhook(user1_sess)
+
+    # admin should be able to create global webhook.
+    admin_sess = api_utils.admin_session()
+    global_webhook_id = create_default_webhook(admin_sess)
+
+    # non-admin should be able to view global webhook.
+    res = bindings.get_GetWebhooks(user1_sess)
+    assert any(w.id == global_webhook_id for w in res.webhooks)
```

Comment:
should we also assert that `len(res.webhooks) > 0` and/or `global_webhook_id > 0`?

Reply:
I believe `any` would raise `AssertionError` is `res.webhooks` is empty. 
 Check for `global_webhook_id ` added